--- nrdp-1.2.a/nrdp.conf	2010-07-24 05:29:42.000000000 +0000
+++ nrdp-1.2.b/nrdp.conf	2015-07-06 10:21:28.012185043 +0000
@@ -1,16 +1,30 @@
-<Directory "/usr/local/nrdp">
-#  SSLRequireSSL
-   Options None
-   AllowOverride None
-   Order allow,deny
-   Allow from all
-#  Order deny,allow
-#  Deny from all
-#  Allow from 127.0.0.1
-#   AuthName "NRDP"
-#   AuthType Basic
-#   AuthUserFile /usr/local/nrdp/htpasswd.users
-#  Require valid-user
-</Directory>
+Alias /nrdp "/srv/eyesofnetwork/nrdp/server"
+
+<Directory "/srv/eyesofnetwork/nrdp/server">
+        Options -Indexes -FollowSymLinks
+        AllowOverride None
+        <LimitExcept POST GET HEAD>
+            Order deny,allow
+            Deny from all
+        </LimitExcept>
+        Order allow,deny
+        # TEST
+        Allow from All
+        # PROD
+        #Allow from 127.0.0.1 X.X.X.X
+        Satisfy Any
 
-Alias /nrdp "/usr/local/nrdp/server"
+        AuthType Basic
+        AuthName "Restricted"
+        AuthEonAuthoritative On
+        AuthEonMySQLSocket /var/lib/mysql/mysql.sock
+        AuthEonMySQLUsername eonweb
+        AuthEonMySQLPassword root66
+        AuthEonMySQLDB eonweb
+        AuthEonMySQLTableSID sessions,users,groupright
+        AuthEonMySQLFieldUID sessions.user_id
+        AuthEonMySQLTableSIDCondition "`sessions`.`session_id`=$session_id AND `sessions`.`user_id`=$user_id AND `users`.`user_id`=$user_id AND `users`.`user_name`=$user_name AND `users`.`group_id`=$group_id AND `groupright`.`group_id`=$group_id AND `groupright`.`tab_6`='1'"
+        AuthEonPageLogin /login.php
+        AuthEonSessionCookies On
+        Require valid-user
+</Directory>
